packages/Tag/src/Tag.js

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import PropTypes from "prop-types";
+import * as sc from "./Tag.styles";
+import * as types from "./types";
+
+export default function Tag(props) {
+  const { children, onRemove, size, onClick, a11yText, avatar, isDisabled, ...moreProps } = props;
+
+  const handleRemove = e => {
+    onRemove();
+    e.stopPropagation();
+  };
+
+  const content = (
+    <>
+      {avatar}
+      <sc.Ellipsis size={size}>{children}</sc.Ellipsis>
+      {onRemove ? (
+        <sc.Delete isDisabled={isDisabled} size={size} data-pka-anchor="tag.remove" onClick={handleRemove} />
+      ) : null}
+    </>
+  );
+
+  if (onClick) {
+    return (
+      <sc.RawButtonTag
+        a11yText={a11yText}
+        isDisabled={isDisabled}
+        onClick={onClick}
+        size={size}
+        {...moreProps}
+        data-pka-anchor="tag"
```

Comment:
I think it's okay to allow the `anchor` to be overridden by `moreProps`.  This allows the `ListBoxWithTags` to label this as `list-box.tag` for example.

Also, the non-button version below allows it with `moreProps` coming last, so best to be consistent.